docs/workflow-controller-configmap.yaml

```diff
@@ -78,6 +81,16 @@ data:
       scope: workflow-list
       url: http://workflows?label=workflows.argoproj.io/completed=true
 
+  # Columns are custom columns that will be exposed in the Workflow List View.
+  columns: |
```

Comment:
this is follow-up to [this PR](https://github.com/argoproj/argo-workflows/pull/10808), correct?

Reply:
Yep! Docs/example for that feature

Reply:
added a comment that this apparently requires Argo v3.5 (I had not realized that earlier) in #11474